src/Type/Definition/BooleanType.php

```diff
@@ -28,7 +27,7 @@ class BooleanType extends ScalarType
      */
     public function serialize($value)
     {
-        return !!$value;
+        return ! ! $value;
```

Comment:
I think we should use `(bool) $value` with a new code style. `! ! $value` looks really weird

Reply:
Agreed, double `!!` looks somewhat magical. TBH I had no idea what it does before.